### ISSUE TITLE
Post details page

### ISSUE THREAD

**Issue by malcolm-kee** [malaysianpaygap/website#9]
depends on #7

**Comment by kychok98** [comment]
Phase 1 not needed

**Comment by malcolm-kee** [comment]
@kychok98 instead of closing features not needed in phase 1, can you reopen them and add the nice to have tag? We can leave them as future enhancements, closing them means we will not review them anymore in future.
